vendors/circom/circomlib/zkey/coefficient.h

```diff
@@ -0,0 +1,37 @@
+#ifndef VENDORS_CIRCOM_CIRCOMLIB_ZKEY_COEFFICIENT_H_
+#define VENDORS_CIRCOM_CIRCOMLIB_ZKEY_COEFFICIENT_H_
+
+#include <stdint.h>
+
+#include <string>
+
+#include "absl/strings/substitute.h"
+
+namespace tachyon::circom {
+
+#pragma pack(push, 1)
+template <typename F>
+struct Coefficient {
+  // if |matrix| is 0, then |a|, otherwise |b| matrix.
```

Comment:
Could you elaborate on what `|a|` and `|b|` are?

Reply:
`a`, `b` are the ones that is used in a R1CS `A * z (Had) B * z = C * z`

Reply:
Ah I see. Would it be possible to include more comments for these `unit32_t` members? I think it would be helpful, especially since `|a|` and `|b|` weren't mentioned anywhere else in this file so I was confused.